src/books.py

```diff
@@ -0,0 +1,35 @@
+from flask import Blueprint, jsonify, request
+from flasgger import swag_from
+
+from src.models import Book
+from src.constants.http_status_codes import HTTP_200_OK, HTTP_404_NOT_FOUND
+
+
+books_bp = Blueprint('books', __name__, url_prefix='/api')
+
+
+@books_bp.get("/books/available")
+@swag_from("./docs/books/available_books.yaml")
+def available_books():
+
+
+    books = Book.query.filter_by(is_available=True)
+
+    if len(list(books)) < 1:
```

Comment:
do you need to convert books to a list before you can get the length? Did you check whether `len` works directly on it or whether books has a method to do so? Does `dir(books)` show anything related to that?

Reply:
No, it doesn't